canonicalwebteam/blog/wordpress_api.py

```diff
@@ -43,16 +43,16 @@ def get_articles(tags, per_page=12, page=1, exclude=None, category=None):
     return process_response(response), total_pages
 
 
-def get_article(tags, slug):
-    url = "".join(
-        [
-            API_URL,
-            "/posts?slug=",
-            slug,
-            "&tags=",
-            "".join(str(tag) for tag in tags),
-        ]
-    )
+def get_article(slug, tags=None, excluded_tags=None):
```

Comment:
Is there any a case when you want tags and excluded_tags at the same time? If you pass both arguments it will output a URL with tags and excluded_tags.

Reply:
Hm, good question. I dont think they need to be mutually exclusive since some articles may have multiple tags?
I would prefer leaving the possibility to add both.

Reply:
Should the tests cover that case then?

Reply:
Good idea! Will implement